### ISSUE TITLE
Missing backtick(`)

### ISSUE THREAD

**Issue by YueLinHo** [progit/progit2#551]
book\02-git-basics\sections\viewing-history.asc line 159

```
<<pretty_format>> lists some of the more useful options that format takes.
```
(for `git log --pretty=format`)

change to?

```
<<pretty_format>> lists some of the more useful options that `format` takes.
```
(on the word **format**)

**Comment by ben** [comment]
Yeah, that's better.

**Comment by ben** [comment]
Closed in #552. :smile:

**Comment by YueLinHo** [comment]
Thanks @ben
